oxidation/crates/parsec_serialization_format/src/parser/mod.rs

```diff
@@ -3,16 +3,16 @@
 mod data;
 mod field;
 mod protocol;
-mod schema;
+#[allow(dead_code)]
```

Comment:
should add a comment to explain why this is needed

Reply:
It isn't needed anymore, I used SerdeAttr::Tag, good catch :+1: